src/pages/workspace/WorkspacesListPage.js

```diff
@@ -169,43 +181,41 @@ function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, u
             }))
             .sortBy((policy) => policy.title.toLowerCase())
             .value();
-    }, [reimbursementAccount.errors, policies, network.isOffline, allPolicyMembers]);
+    }, [reimbursementAccount.errors, policies, isOffline, allPolicyMembers]);
 
     return (
-        <ScreenWrapper>
-            <HeaderWithBackButton
-                title={translate('common.workspaces')}
-                onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
-            />
-            {_.isEmpty(workspaces) ? (
-                <BlockingView
-                    icon={Expensicons.Building}
-                    title={translate('workspace.emptyWorkspace.title')}
-                    subtitle={translate('workspace.emptyWorkspace.subtitle')}
-                />
-            ) : (
-                <ScrollView style={styles.flex1}>{_.map(workspaces, (item, index) => getMenuItem(item, index))}</ScrollView>
-            )}
-            <FixedFooter style={[styles.flexGrow0]}>
+        <IllustratedHeaderPageLayout
+            backgroundColor={themeColors.PAGE_BACKGROUND_COLORS[SCREENS.SETTINGS.WORKSPACES]}
+            illustration={LottieAnimations.WorkspacePlanet}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS)}
+            title={translate('common.workspaces')}
+            footer={
                 <Button
                     accessibilityLabel={translate('workspace.new.newWorkspace')}
                     success
                     text={translate('workspace.new.newWorkspace')}
                     onPress={() => App.createWorkspaceAndNavigateToIt('', false, '', false, !isSmallScreenWidth)}
                 />
-            </FixedFooter>
-        </ScreenWrapper>
+            }
+        >
+            {_.isEmpty(workspaces) ? (
+                <FeatureList
+                    menuItems={workspaceFeatures}
+                    headline="workspace.emptyWorkspace.title"
+                    description="workspace.emptyWorkspace.subtitle"
+                />
+            ) : (
+                <>{_.map(workspaces, (item, index) => getMenuItem(item, index))}</>
```

Comment:
Are you sure you need to wrap this in a fragment? 🤔  I haven't tested removing it, but I assumed you wouldn't need to since this is all inside `IllustratedHeaderPageLayout`

Reply:
Good catch, removed the fragment and retested:

<img width="757" alt="image" src="https://github.com/Expensify/App/assets/47436092/21bd2311-bee9-4ab6-a354-739d517a46e0">